web/src/components/organisms/Asset/AssetList/index.tsx

```diff
@@ -29,6 +36,19 @@ const AssetList: React.FC = () => {
 
   return (
     <AssetListBody
+      commentsPanel={
+        <CommentsPanel
+          collapsed={collapsed}
+          onCollapse={handleCollapsed}
+          emptyText={
+            selectedAsset
+              ? t("No comments.")
+              : t("Please click the comment bubble in table to check comments.")
```

Comment:
Please click the comment bubble in the table to check comments.

Reply:
Fixed.